Jenkinsfile

```diff
@@ -190,6 +190,8 @@ stage("Unit Test") {
           # launch different process for each test to make sure memory is released
           python3 -m pip install --upgrade pytest-xdist
 
+          python3 -m pip install 'black~=22.0,>=22.3'
```

Comment:
Isn't `~=22.0` redundant?

Reply:
I referred to here: https://github.com/huggingface/transformers/blob/main/setup.py#L101
Not sure whether they have some special considerations.